package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@ldapjs/messages",
   "homepage": "https://github.com/ldapjs/messages",
   "description": "API for creating and parsing LDAP messages",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
Please don't change the version number. I'll take care of this when I publish the module.

Reply:
Cool I'll revert